static/app/views/performance/transactionSummary/transactionTags/content.tsx

```diff
@@ -137,10 +132,10 @@ const InnerContent = (
   };
 
   useEffect(() => {
-    if (!decodedTagFromOptions && initialTag) {
+    if (initialTag) {
       changeTagSelected(initialTag);
     }
-  }, [decodedTagFromOptions, initialTag]);
+  }, [initialTag]);
```

Comment:
Cool, so if I'm not mistaken, this is is the core fix and all the other changes are to show loading indicator correctly.

Reply:
It also makes sure the queries aren't called when this is empty by replacing that content with placeholders / loading indicators while tag keys is loading in.